test/utils/client.dart

```diff
@@ -3,63 +3,34 @@ import 'dart:io';
 import 'dart:math';
 
 import 'package:crypto/crypto.dart';
-import 'package:meilisearch/src/http_request_impl.dart';
 import 'package:meilisearch/src/http_request.dart';
-import 'package:meilisearch/meilisearch.dart';
 import 'package:test/test.dart';
 
-late HttpRequest http;
-late MeiliSearchClient client;
-Random random = Random();
+import '../models/test_client.dart';
 
-String get testServer {
-  return Platform.environment['MEILISEARCH_URL'] ?? 'http://localhost:7700';
-}
+HttpRequest get http => client.http;
+late TestMeiliSearchClient client;
+final random = Random();
 
-Future<void> deleteAllIndexes() async {
-  var data = await client.getIndexes();
-  for (var item in data.results) {
-    await item.delete();
+const bool _kIsWeb = bool.fromEnvironment('dart.library.js_util');
+String get testServer {
+  const defaultUrl = 'http://localhost:7700';
+  if (_kIsWeb) {
+    return defaultUrl;
+  } else {
+    return Platform.environment['MEILISEARCH_URL'] ?? defaultUrl;
```

Comment:
Will this new rule work when I run this with docker-compose (using the current docker-compose config)?

Reply:
yes, since on web you can't access environment variables anyway